src/components/navBar/navBar.module.scss

```diff
@@ -0,0 +1,39 @@
+.navBar {
+  width: 100vw;
+  font-size: 1rem;
+  margin: 0 -2em;
```

Comment:
I don't think using negative margins is a good idea.
Let's try to solve the problem from its root.

In this case, we can fix this problem by removing margins from `html` styles, and we won't need the margin in navbar.

https://github.com/Real-Dev-Squad/website-status/blob/6a25cf74187cf413b93418f5a0fab62c1545383d/src/styles/index.scss#L7

Reply:
Yes you are right, but in that case we have to add margins for rest of the UI except navBars (for the mobile view). So I tried it wrapping the rest content in a div and add margins but it was creating some problems. I will try to find a way to do this and as of now I have to redesign (some changes are required only) navBar as it has to look same all across the websites.